### PR TITLE
feat: Support Reconciliation results limit

### PR DESCRIPTION
**Pull request by ddeboer**

### CODE REVIEW

File: packages/network-of-terms-reconciliation/test/server.test.ts [hunk 1]
```diff
@@ -61,6 +61,25 @@ describe('Server', () => {
...
+  it('limits reconciliation API results', async () => {
+    const response = await reconciliationQuery(
+      'https://data.netwerkdigitaalerfgoed.nl/rkd/rkdartists/sparql',
+      {
+        q1: {
+          query: 'art',
+        },
+        q2: {
+          query: 'art',
+          limit: 1,
```
Comment (sdevalk) [inline packages/network-of-terms-reconciliation/test/server.test.ts:73]: Are there restrictions to the `limit`? For example: can I send `limit: -1` or `limit: "blah"`?

**Review by sdevalk — COMMENTED** [review]
Looking good! Just one question :)

File: packages/network-of-terms-reconciliation/test/server.test.ts (hunk 1 above)
Comment (ddeboer) [reply packages/network-of-terms-reconciliation/test/server.test.ts:73]: Good point! Added JSON Schema-based validation of reconciliation requests.

Comment (sdevalk) [reply packages/network-of-terms-reconciliation/test/server.test.ts:73]: Nice!

**Review by sdevalk — APPROVED** [review]
👍

**Pull request merged**